can/message.py

```diff
@@ -202,7 +205,7 @@ def __copy__(self):
         )
         return new
 
-    def __deepcopy__(self, memo):
+    def __deepcopy__(self, memo: dict) -> "Message":
```

Comment:
What does the annotation ` -> "Message"` do?

Reply:
It's a [forward reference](https://mypy.readthedocs.io/en/latest/kinds_of_types.html#class-name-forward-references) for the return type.

You can check out [PEP 563](https://www.python.org/dev/peps/pep-0563/) for more about the proposed Python 4.0 change to address why this is needed.